### ISSUE TITLE
fly pg update not able to resolve image details

### ISSUE THREAD

**Issue by rugwirobaker** [superfly/flyctl#1119]
```
~/Programming/fly/flyctl(refactor/postgres-launcher)$ ./bin/flyctl machines list --app shaun-machine
1 machines have been retrieved.
View them in the UI here

shaun-machine
ID            	IMAGE              	CREATED             	LAST UPDATED        	STATE  	REGION	NAME                 	IP ADDRESS
3287359b647e85	flyio/postgres:14.4	2022-07-27T14:23:07Z	2022-07-27T14:24:01Z	started	ord   	twilight-firefly-4794	fdaa:0:2e26:a7b:9adb:f795:e795:2
```

```
~/Programming/fly/flyctl(refactor/postgres-launcher)$ ./bin/flyctl pg update --app shaun-machine
Acquiring lease on postgres cluster
Resolving cluster roles
  3287359b647e85: leader
Error can't get latest image details for :: Unknown repository
```

**Comment by rugwirobaker** [comment]
This was due to some commands like `flyctl image show` and `flyctl pg create` using `machine.Config.ImageRef` which were removed from flaps instead of using `machine.ImageRef`. It  will be solved in #1021.

**Comment by redjonzaci** [comment]
Can we close this since #1021 was merged?
